### DISCUSSION TITLE
Native custom boot screen

### DISCUSSION THREAD

**Discussion by Starbrush** [OnionUI/Onion#1277] [Ideas]
when we turn on the miyoo mini we have the image "miyoo mini linux games" but it would be very interesting if we could change it to something else, like an image of an onion with "onion os" written on it, or another screen when turning on, having total customization.

• Placing your favorite character
• A representation of your favorite game
• Or anything else because It's your option, You will have the freedom to do whatever you want 
he can accessed through an onion app, like "Custom boot-screen" being simple and practicalwith you being able to place any image you want in the "boot" screen folder

**Comment by schmurtzm** [comment]
Use [Easy Logotweak](https://github.com/schmurtzm/Miyoo-Mini-easy-logotweak).

https://onionui.github.io/docs/apps/logotweak

This logo is in the firmware so changing it represent a little risk of brick (there were very few with Easy Logotweak due to bad manipulation and it was possible to recover from this bricking). But this risk explain why Easy Logotweak is not included in Onion directly.

**Reply by Starbrush** [reply]
I understand, but it's a shame that there is currently no safe way to implement a customizable boot logo in onion os

**Reply by schmurtzm** [reply]
There's already a safe and customizable boot logo in onion os  ;)  [bootscreen.png](https://onionui.github.io/docs/theme-design#blueprint-theme-credit-aemiii91) which is impacted by the themes ;)

**Reply by schmurtzm** [reply]
I agree that it's a shame to not be able to modify the very first bootscreen but it can't be safe as it is in the nand.
However installing Easy Logotweak is quick and easy, users who install it knows that they are making something "not standard".

**Reply by Starbrush** [reply]
Oh btw, thank you for answering me! 😁
